pkg/noun/manage.c

```diff
@@ -2165,7 +2181,51 @@ u3m_migrate(u3v_version ver_w)
 
   /* finally update the version and commit to disk */
   u3H->ver_w = ver_w;
-  /* extra assurance we haven't corrupted the loom before writing to disk */
+  assert(u3m_save_snapshot() == c3y);
+}
+
+c3_o
+u3m_save_snapshot(void)
+{
+  assert(u3_pma);
```

Comment:
This (and all others were c3 is in scope) should be `c3_assert()`.

Reply:
Fixed.